java/common-job/src/main/java/sleeper/job/common/DescribeClusterException.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2022-2023 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package sleeper.job.common;
+
+public class DescribeClusterException extends Exception {
```

Comment:
Is there a benefit to using a checked exception rather than just letting it throw all the way out? It does seem like we want the whole thing to terminate if it fails?